### PR TITLE
P0014 clean

### PR DESCRIPTION
**Pull request by hemantkumargangwal**

### CODE REVIEW

**Review by rahul1choudhary — CHANGES_REQUESTED** [review]
Please do discussed changes

File: apps/proxy/src/app/register/register.component.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<div style="height: 100vh; background-color: #fafafa">
```
Comment (rakeshshrivas) [inline apps/proxy/src/app/register/register.component.html:1]: use class

File: apps/proxy/src/app/register/register.component.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    <mat-card
+        class="h-75 container outline-card responsive-card py-5 --border-common-radius"
+        style="box-shadow: rgba(0, 0, 0, 0.25) 0px 25px 50px -12px; background-color: #e6e6fa; width: 40%"
```
Comment (rakeshshrivas) [inline apps/proxy/src/app/register/register.component.html:4]: remove style

File: apps/proxy/src/app/register/register.component.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    >
+        <h2 class="d-flex justify-content-center">Register</h2>
```
Comment (rakeshshrivas) [inline apps/proxy/src/app/register/register.component.html:6]: text-center

File: apps/proxy/src/app/register/register.component.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+        <div style="margin-top: 50px">
+            <div [formGroup]="registrationForm" proxyMarkAllAsTouched (valid)="submit()">
+                <div class="d-flex gap-3 mt-20">
```
Comment (rakeshshrivas) [inline apps/proxy/src/app/register/register.component.html:9]: remove margin mt-20 and add class `mt-4`

**Review by rakeshshrivas — CHANGES_REQUESTED** [review]

**Review by rakeshshrivas — CHANGES_REQUESTED** [review]

**Review by rakeshshrivas — CHANGES_REQUESTED** [review]

File: apps/proxy/src/app/register/register.component.html [hunk 5]
```diff
@@ -89,7 +87,7 @@ <h2 class="d-flex justify-content-center">Register</h2>
     let-formControl="formControl"
     let-patternError="patternError"
 >
-    <mat-form-field appearance="outline" style="width: 50%">
+    <mat-form-field appearance="outline" style="width: 100%">
```
Comment (rakeshshrivas) [inline apps/proxy/src/app/register/register.component.html:90]: class use class `w-100`

File: apps/proxy/src/styles.scss [hunk 6]
```diff
@@ -95,12 +95,12 @@ canvas {
     }
 }
 .register-user-details {
-    width: 50%;
+    width: 100%;
     margin-top: 5px;
+    height: 61px;
```
Comment (rakeshshrivas) [inline apps/proxy/src/styles.scss:100]: remove height

File: apps/proxy/src/styles.scss [hunk 7]
```diff
@@ -95,12 +95,12 @@ canvas {
     }
 }
 .register-user-details {
-    width: 50%;
+    width: 100%;
```
Comment (rakeshshrivas) [inline apps/proxy/src/styles.scss:98]: remove css

**Review by rakeshshrivas — CHANGES_REQUESTED** [review]

**Review by rakeshshrivas — APPROVED** [review]

**Review by rahul1choudhary — APPROVED** [review]

**Pull request merged**